tests/unit_tests/test_tracks.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
+
+import numpy as np
+import openmc
+import pytest
+
+from tests.testing_harness import config
+
+
+@pytest.fixture
+def sphere_model():
+    openmc.reset_auto_ids()
+    mat = openmc.Material()
+    mat.add_nuclide('Zr90', 1.0)
+    mat.set_density('g/cm3', 1.0)
+
+    model = openmc.Model()
+    sph = openmc.Sphere(r=25.0, boundary_type='vacuum')
+    cell = openmc.Cell(fill=mat, region=-sph)
+    model.geometry = openmc.Geometry([cell])
+
+    model.settings.run_mode = 'fixed source'
+    model.settings.batches = 1
+    model.settings.particles = 100
```

Comment:
```suggestion
    model.settings.batches = 2
    model.settings.particles = 50
```

I would be tempted to simulate 2 batches just to test accessing the tracks from a separate batch

Reply:
addressed in the commits